src/triage/risklist/__init__.py

```diff
@@ -0,0 +1,171 @@
+from triage.component.results_schema import upgrade_db
+from triage.component.architect.entity_date_table_generators import EntityDateTableGenerator, DEFAULT_ACTIVE_STATE
+from triage.component.architect.features import FeatureGenerator
+from triage.component.architect.builders import MatrixBuilder
+from triage.component.catwalk.predictors import Predictor
+from triage.component.catwalk.utils import filename_friendly_hash
+from triage.util.conf import dt_from_str
+
+from collections import OrderedDict
+import json
+import re
+
+
+def get_required_info_from_config(db_engine, model_id):
+    """Get all information needed to make the risk list from model_id
+    Args:
+            db_engine (sqlalchemy.db.engine)
+            model_id (int) The id of a given model in the database
+
+    Returns: (dict) a dictionary of all information needed for making the risk list
+
+    """
+    get_experiment_query = """
+        select experiments.config, matrices.matrix_metadata, matrix_uuid
+        from model_metadata.experiments
+        join model_metadata.experiment_matrices using (experiment_hash)
+        join model_metadata.matrices using (matrix_uuid)
+        join model_metadata.models on (models.train_matrix_uuid = matrices.matrix_uuid)
+        where model_id = %s
+    """
+    results = list(db_engine.execute(get_experiment_query, model_id))
```

Comment:
You might just want to use `first()` here:

```suggestion
    result = db_engine.execute(get_experiment_query, model_id).first()
```

Reply:
@thcrock